week-1/L-functions-nested/exercise.js

```diff
@@ -1,8 +1,28 @@
-function createPercentage(sts, mts) {
-var sumT = sts + mts
-var percSts = sts/sumT * 100;
-var percMts = mts/sumT * 100;
-    return percSts, percMts;
+var mentor1 = "Daniel";
+var mentor2 = "Irina";
+var mentor3 = "Mimi";
+var mentor4 = "Rob";
+var mentor5 = "Yohannes";
+
+
+// the percentage function//
+function createPercentages(sts,mts) {
+var percSts = (mts/(mts+sts))* 100;
+var percMts = (sts/(mts+sts))* 100;
+displayPercentages(percSts,percMts);
  }
- function greeting ()= createLongCreeting('Daniel', 30);
- console.log(greeting);
\ No newline at end of file
+
+function displayPercentages(percSts,percMts){
+console.log("Percentage of Students " + Math.round(percSts) + "%");
+console.log("Percentage of mentors " + Math.round(percMts) + "%");
+}
+
+function displayGreetings(name){ 
+    console.log("Hello " + name.toUpperCase() + "!");
```

Comment:
inside a function we use return not console.log. think how you can use nested functions to get the result.